### PR TITLE
feat(response): add cookie header

### PR DESCRIPTION
**Pull request by dantol29**
I added `response.setHeader("Set-Cookie", "meal=salad; Expires=Wed, 09 Jun 2024 10:18:14 GMT; Path=/; Secure; HttpOnly")`

Now when we send `Set-Cookie` header browser stores this information! You can see it in developer console 
image

And then browser sends back saved cookies
image

### CODE REVIEW

**Comment by dantol29** [comment]
I am currently connecting PostgreSQL database to be able to use DELETE

**Comment by 552020** [comment]
Tests are failing!

**Comment by 552020** [comment]
@dantol29 tests are failing ... :-(

**Comment by dantol29** [comment]
@552020 @lmangall Ready to be merged!

File: var/localhost:8080/index.html
Comment (lmangall) [inline var/localhost:8080/index.html:1]: 🙉🙊🙈

Comment (lmangall) [reply var/localhost:8080/index.html:1]: But the js is nice!

**Review by lmangall — DISMISSED** [review]
LGTM

**Review by 552020 — APPROVED** [review]
🥗

**Review by lmangall — APPROVED** [review]
🍪

**Pull request merged**